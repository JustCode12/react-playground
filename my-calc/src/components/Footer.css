.Footer{
    color: blue;
    background-color: blueviolet;
}