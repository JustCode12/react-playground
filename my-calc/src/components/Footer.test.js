import React from 'react';
import Footer from './Footer';
import {shallow, mount} from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Footer component', () => {
    const wrapper = shallow(<Footer/>);
      it('should render component', () =>{
        const wrapper = shallow(<Footer/>);
        expect(wrapper).toMatchSnapshot();
      });
      it('should render Footer', ()=>{ 
          const wrapper = mount(<Footer/>);
          expect(wrapper.find(Footer).length).toBe(1);
      });
      it('should render Footer', ()=>{ 
        const wrapper = mount(<Footer/>);
        expect(wrapper.html()).toEqual('<div class="Footer">This is Footer.</div>');
    });

})
