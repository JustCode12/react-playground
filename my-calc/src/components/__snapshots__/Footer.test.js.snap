// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render component 1`] = `ShallowWrapper {}`;
